### PR TITLE
systemd: Service start and stop buttons redesign

### LINKED ISSUE #9430: service detail page fail a bit ungracefully when used as non-admin, redesign buttons
Tried to stop Docker with a non-admin account and got this error box.
screenshot from 2018-06-19 18-12-36

I think it would be better to make the actions insensitive instead, like we do in other places.

### PR DESCRIPTION
**Pull request by Gundersanne**
Fixes #9430

### CODE REVIEW

**Comment by Gundersanne** [comment]
Admin:
screeny-enabled

Non-admin:
screeny

File: pkg/systemd/init.js [hunk 1]
```diff
@@ -510,9 +510,11 @@ $(function() {
     var template_template = $("#service-template-tmpl").html();
     mustache.parse(template_template);
 
-    var unit_actions = [                          // <method>:<mode>
+    var unit_primary_actions = [                          // <method>:<mode>
         { title: _("Start"),                 action: 'StartUnit' },
-        { title: _("Stop"),                  action: 'StopUnit' },
+        { title: _("Stop"),                  action: 'StopUnit' }
```
Comment (martinpitt) [inline pkg/systemd/init.js]: Just a hint: Personally I'm a big fan of trailing commas in such lists, as this avoids this kind of diff noise. We use these in plenty of places, so feel free to do it here, too.

**Comment by martinpitt** [comment]
This seems to break the [testIpaUnqualifiedUsers](https://fedorapeople.org/groups/cockpit/logs/pull-9522-20180703-113434-4705019a-verify-ubuntu-1604/log.html#202) test case, which is also looking for that service button. This needs to be adjusted accordingly.

**Comment by martinpitt** [comment]
[testIpa](https://fedorapeople.org/groups/cockpit/logs/pull-9522-20180703-113434-4705019a-verify-ubuntu-stable/log.html#201) fails, too.

**Comment by Gundersanne** [comment]
@martinpitt Patched the tests as well.

**Pull request merged**